web/blob/templates/status.html

```diff
@@ -47,7 +47,12 @@
           {{range $pool}}
             <tr>
               <td>
-                <a href="{{.URL | globalURL}}">{{.URL}}</a>
+              <a href="{{.URL | globalURL}}">{{.URL.Scheme}}://{{.URL.Host}}{{.URL.Path}}</a><br>
+                {{range $label, $values := .URL.Query }}
+                {{range $i, $value := $values}}
```

Comment:
I'd say indent the nested template lines as well, and the span within. We don't care what the generated HTML code looks like, just want to be able to see the nested structure in the source code here.